src/txmempool.h

```diff
@@ -672,10 +672,10 @@ class CTxMemPool
 class CCoinsViewMemPool : public CCoinsViewBacked
 {
 protected:
-    CTxMemPool &mempool;
+    const CTxMemPool &mempool;
 
 public:
-    CCoinsViewMemPool(CCoinsView *baseIn, CTxMemPool &mempoolIn);
+    CCoinsViewMemPool(CCoinsView *baseIn, const CTxMemPool &mempoolIn);
```

Comment:
It is not really important to get every white space "correct" but you can read about the general preferred code style over here: https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#developer-notes